Kontent.Ai.Management.Tests/Kontent.Ai.Management.Tests.csproj

```diff
@@ -33,6 +33,25 @@
 		<ProjectReference Include="..\Kontent.Ai.Management\Kontent.Ai.Management.csproj" />
 	</ItemGroup>
 
+	<ItemGroup>
+		<AssemblyAttribute Include="Kontent.Ai.Management.Attributes.SourceTrackingHeader">
+			<!--
+			<_Parameter1>My.Module</_Parameter1>
+
+			<_Parameter2>1</_Parameter2>
+			<_Parameter2_IsLiteral>true</_Parameter2_IsLiteral>
+
+			<_Parameter3>2</_Parameter3>
+			<_Parameter3_IsLiteral>true</_Parameter3_IsLiteral>
+
+			<_Parameter4>3</_Parameter4>
+			<_Parameter4_IsLiteral>true</_Parameter4_IsLiteral>
+
+			<_Parameter5>beta</_Parameter5>
+			-->
```

Comment:
Why there are these comments?

Reply:
It is just an alternative way how to use the new attribute with custom values without modifying `assemblyinfo.cs`.

Reply:
Yes, but it is commented out - so it should be uncommented and tested propagation in the test or deleted right?

Reply:
No, because I want to test the default configuration (without custom parameters). Unfortunately, you can't test both in one assembly

Reply:
Alright -> add information about tracking and possibilities (AsseblyInfor/CSproj with a link to this part.)